plugins/csv/src/main/java/org/itsallcode/whiterabbit/plugin/csv/DirectoryStreamProvider.java

```diff
@@ -0,0 +1,21 @@
+package org.itsallcode.whiterabbit.plugin.csv;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class DirectoryStreamProvider implements OutStreamProvider {
+
+    private final String outPath;
+
+    DirectoryStreamProvider(String outPath) {
```

Comment:
```suggestion
    DirectoryStreamProvider(Path outPath) {
```
Using Path instead of String makes it clear which type should be used.